flows/conditional-flows/index.md

```diff
@@ -0,0 +1,220 @@
+m+AN148+2xejdY8dQzL0Tzek/sPltdvfubs+95p/fSrcM/P/oBTqaNwPn1t985IZp3o/c6V2reXaFqt/9ZzEqo3ESAM7pvLLbc6jqwLVeZOqKbVgez9fak2SnA+ZdD3ciBX98pwE166NHGrDnzmuqP2J3o2qll0wKc/81Val1p70WR7eUkJ3WCa3Wi6/cEJzth22dVn5MLTu6A6Ycav127fuMCj6ZnvD47mnbrc8v7DyVYVS/dQd3p6u3fgXT3ptp1Kay5A7N9DdpnjarbZVT3xuy3Yq9PZXbUoYzWtW+3UkEnrd/YvpJ2rHNfKPyyZ59/OVGmLw+atgHDqX7u32KgLw+unb70uL6rvyW3znZfilWv+8FabdMFNve3Ld3fiL8OfzotCLnApml3lcjpRq79Mru8P6I2b/6CxDbbvZduvp8Ap+krr7o2Vq/XlHa7RT9mJVRuIsAZX31tVttO6bTfUJwqcw86aFqjBLY+a4Dzl/dPnK6+U4Brpb9MmmqTdlCz7XR/UVp5ke31JCfTTnQhTnCyE7Z9VvU5+SNffl+w/aqVfhuFPN1TpxEDv2/a7VhVr7+Niy65vHHJmLHRvWx+vf07kLpE4pf54U/T9hv+/HcXxfbbf/rWviZnFZ60rmvfbqUuu9vP2anLpmqjaQUMfzmVpR2/7fr9JyNtwPCX+3DVuub8O+8duESqfzWyrVCn+zXTtqPLlfoy4u+7RsFbbdMFOL/MPgVrp9NeqztXadref337XUOjW36ZXd6fTzt32WXsU6aa7jfAtQtr9pJur2YlVG4iwKWoD/3m2w4Mbztfnvpqs3OknWDcsjfcdGtzut0fhZvvN8DpcpWbd5c0/QBnT47dqmXTApz9g3D3J9nli2w/Jznpn+hCneBkJ2z7rOpz8oOV6y82hHfTb9zBU23dvSSa7nRvjGvXLuDZvwPpB7gXXpoWTfuj0+6eTLsu6W5vcJdI0/pxVaxr326lG7lVQLO6E7ir95dTWdrx267f3d+saRsw5Ko1GxJlbl3uS4YC15tvvRNN25EyjbRpXrcAuH3UvL74tNrmcAS4G24cOq85R42+NLENu7w/n3bussv476Vr02+Asw8Xtmrbj1kJlZsIcCleOOri6IP1v3G4e2p0Q7Ur0/yvDj+qOZ92Emz3R+Hm/bBoT1y2g9k/Au2PP+/ClAtw7iZRfx2a15OzflmaapcW4Pz16SlczeukbZcvsv2e5GToE5zshG2fVX1WNjhpFM1+rt30G/fZ++3c5XT/Psk00/bD1/4dSD/A6V4dv96dsFyZfo7Er3dB9a35QyNLdr/d8hpVsPtSNuvat1vpbjuxoy3+MUv13QY4jRrbMvd3YQOGdA8H+GVSI2ru4Rt/XQok/t+Z5u0tL2qX5wicG/2z9bbM1vvz9tyl6XbvpZv3nz63r7WbAKcrBH695v/69PNRsLP72KtZCZWbCHAt1A2hroM6/fAmdbOlbTNn7vxmvebb/VFIN2rgOpI9cdkOZv8IVq/dGNu+vqXpX/uEldVfZyvVzga4tPXp6STbpuiGOMnJtPuF+rETtn1W9XmlBad2B+h2/caWu2DV6udD/OXS9sNp/w6kvYRq9809pd2qPu2hDF97Iiirde3b7dSlSX3GetLZ9VHbV7oJcDrxq1xXPXS/mr36YAOGW4//sJvTPZzgX7Z3++X/jpl7OlsPPeieMTfqnWeAc/PSfUmzgwdWW2fPXZ3eS3+b7uEp+1o7Bbj3Fn4QzWtkT0/E2n222+vVrITKTQS4DmqIO+0maF/dDK1752x5t2p0QCN8tjyLuoRlf/vNVwcEXXbSI+y2rhf1h6HLUaHWl7ehTnKh7YRtP9yG7jeh1WXQ51+cmih3apTQ3iht63Vi82/2Lrv07XTP+f0FzUBgv4yrrJsAJ/3/pUdfCvzjrg0Ybj12HX6de3BB6vabtPb6yR63TbeNPC+hOnWVST/LowcB9FSs3YZd3p+3AU62ey/dMv69rfa16vdX2wU46Qd2hUT3KxGy3/OuMyuhchMBDmspJzmsqvTt4uiebrblZdQ+LOd+X822q6NZCZWbCHBYSznJYVWlbxdHjVzrIQZbXjY1EqewptsU7vrLvc1L0vo1Btu2jmYlVG4iwGEt5SSHVZW+jcOhfkDY/Td8uves061FdTIroXITAQ5rKSc5rKr0bcR8zUqo3ESAw1rKSQ6rKn0bMV+zEio3EeCwlnKSw6pK30bM16yEyk0EOKylnOSwqtK3EfM1K6FyEwEOayknOayq9G3EfM1KqNxEgMNaykkOqyp9GzFfsxIqNxHgEAtkJ2x7xLLYCdsesSxmJVRuIsAhFshO2PaIZbETtj1iWcxKqNxEgEMsiB9t22G7cgK7DGIZpG9jVe2mb1tC5SYCHGJB3PvZPtuVE9hlEMsgfRur6mf79tuu3JFQuamSAU7YNxmxyG77dKftwi2xyyIWWfo2VtVPtu+yXbgrQuWmygY4sX3H7s/f5O2JNx2xKGr4vZvRCQt9G4tuf307uT7Eoqi+3cvImyNUbqp0gAMAAAAoEqFyEwEOAAAAICdC5SYCHAAAAEBOhMpNBDgAAACAnAiVmwhwAAAAADnw2Wf7Gh8sW2OLe4IABwAAAJADWz76pLFuwxZb3BNBAtzuPXttMQAAAAB4KDPt6+MnSHz6DnC7du9hFA4AAACgDTt37g6al/oOcGLFyvWNNes322IAAAAAaIS/5SxIgBO6KW/Fqg22GAAAAKC2uJG3HTt6+6+3WhEswIkNmz6KdnLVmo3RDgMAAADUDT1tqgcWlIlCj7w5ggY4x7ZPtkeXVd2OIyIiItZFXZXU06ahHlhIY1gCHAAAAAAMHwQ4AAAAgJJBgAMAAAAoGQQ4AAAAgJJBgAMAAAAoGQQ4AAAAgJJBgAMAAAAoGQQ4AAAAgJJBgAMAAAAoGQQ4AAAAgJJBgAMAAAAoGQQ4AAAAgJJBgAMAAAAoGf8/JKKK6wZer/YAAAAASUVORK5CYII=>
```

Comment:
Proc ty obrazky jsou inline? To generovalo AI?
Nikde v dokumentaci to nepouzivame, vzdycky to je jako samostatne soubory.
Takhle to neni ani videt v preview toho markdownu.

Reply:
Protoze Zuzka to mela v Google docu a ten ma nove export do markdownu primo a takhle to udela. (nechtelo se mi to stahovat jako web stranku). To ze to neni videt v preview je zvlastni, nevidim duvod proc by to nemelo fungovat :/

Reply:
@ErikZigo dal jsem tam normalni obrazky